pkgs/development/python-modules/fpdf/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, writeText, buildPythonPackage, fetchPypi }:
```

Comment:
Where are `stdenv` and `writeText` used?

Reply:
Aah, I just pressed the big green button. But indeed, don't add parameters that are not used.

Reply:
Sorry, I copied the template from an older Python-package PR and forgot to remove those.

Reply:
Just make sure to remove them with the next update. Not important right now.